testing/utils.go

```diff
@@ -165,7 +165,7 @@ func (tp *TestPDS) RequestScraping(t *testing.T, b *TestBGS) {
 	t.Helper()
 
 	c := &xrpc.Client{Host: "http://" + b.Host()}
-	if err := atproto.SyncRequestCrawl(context.TODO(), c, tp.RawHost()); err != nil {
```

Comment:
did the requestCrawl endpoint change? or is this just a weird artifact?

Reply:
It's effectively the same but it supports cborgen now as well as json afaict.

Reply:
Or at least it uses an input struct instead of a `map[string]interface` for marshalling the request body (and that struct supports JSON hints as well as CBORGEN hints)